spec.bs

```diff
@@ -4803,6 +4799,10 @@ a [=reporting context=] |reportingContext|, and [=origins=] |origin| and |aggreg
               |origin|, |aggregationCoordinator| and |reportingContext|.
       : <a spec="private-aggregation-api" for="scoping details">get debug scope steps</a>
       :: An algorithm that returns |debugScope|.
+    : <a spec="private-aggregation-api" for="PrivateAggregation">should perform default
+        contributeToHistogramOnEvent() processing</a>

```

Comment:
if we want this algorithm to cover the PA's contributeToHistogramOnEvent(), then it needs to be updated, like [its step 8](https://patcg-individual-drafts.github.io/private-aggregation-api/#dom-privateaggregation-contributetohistogramonevent), which throws when the event is not "reserved.xx", but it should also allow events that does not start with "reserved.".

Reply:
oh never mind, seems [=determine whether to perform default contributeToHistogramOnEvent() processing=] never returns true, so it will run after step 3 of [[=contributeToHistogramOnEvent=]](https://patcg-individual-drafts.github.io/private-aggregation-api/#dom-privateaggregation-contributetohistogramonevent)

Reply:
yeah probably explain that the algorithm will never return true or never perform default xxx on the next line, or in the algirhtm's definition, something like `they return a [=boolean=] (only false)`